src/chains.ts

```diff
@@ -13,39 +13,40 @@ const defaultL2Addresses = {
   simulateTxAccessorAddress: "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
 };
 
+const blast = {
+  id: 81457 as const,
+  name: "Blast",
+  network: "blast",
+  nativeCurrency: {
+    decimals: 18,
+    name: "Ether",
+    symbol: "ETH",
+  },
+  rpcUrls: {
+    default: {
+      http: [],
```

Comment:
dont we need rpc urls?

Reply:
it should work with the one stored in metamask!